stock-managment/src/main/java/org/dhis2/android/rtsm/ui/home/HomeViewModel.kt

```diff
@@ -80,6 +80,9 @@ class HomeViewModel @Inject constructor(
     private val _deliveryTo = MutableStateFlow<String?>(null)
     val deliveryTo: StateFlow<String?> get() = _deliveryTo
 
+    private val _test = MutableStateFlow("")
+    val test: StateFlow<String> get() = _test
```

Comment:
I don't understand the purpose of this property, could you add a more meaningful name?

Reply:
This as been solved.